java/java.lsp.server/vscode/package-lock.json

```diff
@@ -1,1232 +1,8 @@
 {
 	"name": "apache-netbeans-java",
 	"version": "0.1.0",
-	"lockfileVersion": 2,
```

Comment:
This seems like a newer version of NPM was used - @dbalek OK ?

Reply:
Fixing in 40a21f6